tox.ini

```diff
@@ -176,7 +176,9 @@ skipsdist = True
 usedevelop = True
 deps =
     ipfshttpclient==0.8.0a2
-commands = {toxinidir}/scripts/generate_ipfs_hashes.py --check {posargs}
+commands = 
+    python -m aea.cli hash all --check
+    python -m aea.cli hash all --packages-dir=./tests/data/ --check
```

Comment:
I've removed the test package definitions from the cli tool and update the method to find packages in a given directory so we can provide test data as packages dir path.